### ISSUE TITLE
Can't access the GUI on cables.

### ISSUE THREAD

**Issue by Lordkek69** [PrinceOfAmber/Storage-Network#569]
Minecraft Version: 1.20.1

Forge Version: NeoForge 1.20.1

Mod Version: 1.20.1-1.11.3 and Alpha

Single Player or Server: Singleplayer

Describe problem (what you were doing; what happened; what should have happened):

I can't access the GUI of the cables so basically my upgrades can't function properly and i don't see any errors on the logs.
I tried the most downloaded version and the (currently) Alpha on Neoforge.

Log file link:

Video/images/gifs (direct upload or link):

**Comment by Lothrazar** [comment]
yeah only some cables have a GUI, some dont thats just the way it is.   example Link cable, no GUI.  Export cable has one though.  Latest 1.20.1 version is https://www.curseforge.com/minecraft/mc-mods/simple-storage-network/files/8178993
